walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -25,9 +25,25 @@
   @Test
   public void testGreeting() {
     Greeter greeter = new Greeter();
-
     String greeting = greeter.greet("Ada");
+    assertEquals("Hello Ada", greeting);
+  }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+    String greeting = greeter.greet("   Ada   ");
+    // Whitespace should be trimmed
+    assertEquals("Hello Ada", greeting);
```

Comment:
No yoda equality. In other words, do not do things like:
"if (5 == num)". Instead prefer:
"if (num == 5)". Its more natural and easier to read. 

So this should be:
assertEquals(greeting, "Hello Ada");

If you want to maintain assertEquals(expected, got) then just toss "Hello Ada" in a variable of its own. So you could have:

String gotGreeting = greeter.greet("   Ada   ");
String expectedGreeting = "Hello Ada";
assertEquals(expectedGreeting, gotGreeting);